overlord/snapstate/backend_test.go

```diff
@@ -206,6 +206,14 @@ func (f *fakeSnappyBackend) RemoveSnapData(info *snap.Info) error {
 	return nil
 }
 
+func (f *fakeSnappyBackend) RemoveSnapSharedData(info *snap.Info) error {
+	f.ops = append(f.ops, fakeOp{
+		op:   "remove-snap-shared-data",
+		name: info.MountDir(),
```

Comment:
My comment was based on just the name: ... MountDir connection but I see that other tests use this as well.